config/production.js

```diff
@@ -1,3 +1,18 @@
 module.exports = {
-
-};
\ No newline at end of file
+  provisioner: {
+    id:                   'aws-provisioner2',
+    workerTypeTableName:  'AWSWorkerTypesV2',
+    publishMetaData:      'false',
+    statsComponent:       'aws-provisioner2',
+    awsKeyPrefix:         'aws-provisioner2-managed:',
```

Comment:
the `2` has to go when we go into production...
Granted we might want to come up with a non-conflicting awsKeyPrefix... perhaps `provisioner-managed:`